doc/getting_started/getting_started.rst

```diff
@@ -32,8 +32,9 @@ These tutorials rely on the master branch of MoveIt2, which requires a build fro
 You will need to have a `colcon <https://docs.ros.org/en/foxy/Tutorials/Colcon-Tutorial.html#install-colcon>`_ workspace setup: ::
 
   mkdir -p ~/ws_moveit2/src
-  cd ~/ws_moveit2/src
-
+  cd ~/ws_moveit2/src 
```

Comment:
Formatting issue  in `cd ~/ws_moveit2/src  `: there is a trailing whitespace, which causes CI to fail (see https://github.com/ros-planning/moveit2_tutorials/actions/runs/785001092). Can you please remove it?

Also as a general comment please try to use pre-commit so you don't have formatting issues (see how to use pre-commit in the README https://github.com/ros-planning/moveit2_tutorials#formatting-and-style).

Reply:
Spacing is removed. Thanks @vatanaksoytezer will use `pre-commit` next time.

Reply:
All space-trailing removed. 
(did not make a local clone to use `pre-commit` , hence easily missed when making quick browser edits).